allegro.cpp

```diff
@@ -3451,7 +3451,7 @@ void Alg_seq::beat_to_measure(double beat, int *measure, double *m_beat,
             prev_den = time_sig[tsx].den;
         } else {
             m = m + (beat - prev_beat) / bpm;
-            *measure = m;
+            *measure = static_cast<int>(m);
```

Comment:
This is really ancient C code. I much prefer (int) m since I still can't keep all the different cast flavors straight.

Reply:
`std::floor()` does not return an integer type, so it doesn't really help with type safety and will not resolve the compiler warning. Additionally, `std::floor()`'s behavior is only more correct if it is valid for `m` to take on negative values. If that is the case, then it is a valuable addition; otherwise, not so much.

Reply:
OK, I think static_cast<int> is ugly and confusing compared to int(m) or (int) m, but I agree static_cast<int> is arguably more correct, so it's OK with me.

Reply:
After my most recent commit, it is a C-style cast like you requested above. Would you like it to stay that way, or should I use `static_cast`? `static_cast` does some compile-time checks C-style casts don't, so it has safety benefits, but yeah it is kind of ugly lol

Reply:
Keeping the C-style cast, resolving this comment